9/problem9.rb

```diff
@@ -0,0 +1,66 @@
+require 'date'
+require 'time_difference'
+
+puts "Enter your Birth date:(DD/MM/YYYY)"
+
+b_date=gets.chomp!
```

Comment:
No need to use bang operator here because you are assigning value returned from chomp to b_date.
Bang operator will also change (and save) gets's value. In this situation this makes no difference but this could lead to errors on some unknown situations.